source/dataset-CN/geo3al/plot_geo3al.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+data=geo3al.gmt
+cpt=geoage.cpt
+lengsize=0.381c
+
+gmt begin geo3al png,pdf
+    gmt set FONT_ANNOT_PRIMARY 10 MAP_TICK_LENGTH_PRIMARY 0.08
+    # plot coastlines
+    gmt coast -R70/150/13/55 -JM22c -Baf -Df -G255 -BWsNe
+    # plot the geology map
+    # -aZ="GLG": using the "GLG" property as the Z value
+    # -G+z -C$cpt: the color is determined by the Z value and the CPT file
+    gmt plot $data -C$cpt -aZ="GLG" -G+z
+
+    # plot rock types using diffenrent patterns
+    # -aI="TYPE": using the "TYPE" properties as ID
+    # -S"-Iv": only keep data segments that matches "-Iv"
+    # v means XXX rocks, i means XXX rocks, w means XXX rocks
```

Comment:
我建议成三行依次解释`-S"-Iv"`, `-S"-i"`, `-S"-Iw"`.
```suggestion
    # -S"-Iv": only keep data segments that matches "-Iv" (v means XXX rocks)
    # -S"-Ii": only keep data segments that matches "-Ii" (i means XXX rocks)
    # -S"-Iw": only keep data segments that matches "-Iw" (w means XXX rocks)
```